src/docs/product/integrations/index.mdx

```diff
@@ -13,6 +13,11 @@ description: "Learn more about the wide variety of apps and services integrated
 
 Sentry integrates with your favorite apps and services. Each integration offers features that help track and triage your errors.
 
+## Integration Platform
+Sentry’s Integration Platform provides a way for external services to interact with the Sentry SaaS service using the REST API and webhooks. Integrations utilizing this platform are first-class actors within Sentry, and you can build them for [public](/product/integrations/integration-platform/#public-integrations) as well as [internal](/product/integrations/integration-platform/#internal-integrations) use cases.
```

Comment:
@MimiDumpling Technically the integration platform isn't just for SaaS but maybe we don't need to care about that detail.

Reply:
No prob. We can remove that part. I'll update that on the Integration Platform index page too.